src/cascadia/TerminalApp/TerminalAppLib.vcxproj

```diff
@@ -27,15 +27,13 @@
   <Import Project="..\..\..\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
   <Import Project="$(OpenConsoleDir)src\cppwinrt.build.pre.props" />
   <Import Project="..\..\..\packages\Microsoft.Toolkit.Win32.UI.XamlApplication.6.0.0\build\native\Microsoft.Toolkit.Win32.UI.XamlApplication.props" Condition="Exists('..\..\..\packages\Microsoft.Toolkit.Win32.UI.XamlApplication.6.0.0\build\native\Microsoft.Toolkit.Win32.UI.XamlApplication.props')" />
-

```

Comment:
nit: it'd be nice if we could keep the extra empty lines here and there. Makes it more legible.

Reply:
ah something keeps automagically keeps removing the spaces for me and it's on me to double check that I'm not pushing these in - maybe it's runformat? I'll revert it back 👍

Reply:
I think I ran into that problem before too, but I forget how. Nowadays, I just edit it in VS Code. Then when I switch to VS I get a prompt to reload all. That might work? idk 🤷‍♂️